### PR TITLE
micassess 1.0.0, pep8 fixes

### LINKED ISSUE #11: update micassess to v1.0.0
https://github.com/cianfrocco-lab/Automatic-cryoEM-preprocessing

### PR DESCRIPTION
**Pull request by azazellochg**
Close #11

### CODE REVIEW

**Review by delarosatrevin — COMMENTED** [review]
Maybe add a validation that the models exist?

**Comment by delarosatrevin** [comment]
Maybe an option to display BAD micrographs from the analyze results?

**Comment by delarosatrevin** [comment]
@azazellochg probably this is not related to your changes, but the micassess protocol is not working in streaming. I mean, for the case the input is a closed set, but the batch size is 16, I would expect that good/bad is being populated in the output before the completion of the protocol.

File: README.rst [hunk 1]
```diff
@@ -72,13 +72,13 @@ python with conda ones. One example of this could can be seen below but
...
 
-*CRYOASSESS_ENV_ACTIVATION* (default = conda activate cryoassess-0.2.0):
+*CRYOASSESS_ENV_ACTIVATION* (default = conda activate cryoassess-1.0.0):
 Command to activate the cryoassess environment.
 
 The deep-learning models can be downloaded from
 `authors' website <https://cosmic-cryoem.org/software/cryo-assess/>`_ and set with the following variables:
 
-*CRYOASSESS_MODEL_MIC* (default = software/em/cryoassess-models/micassess_051419.h5)
+*CRYOASSESS_MODEL_MIC* (default = software/em/cryoassess-models)
```
[the remark this answers is not in the document]
Comment (azazellochg) [reply README.rst:81]: You are right, we can only have one var pointing to a folder since we are removing support for older versions.

**Comment by azazellochg** [comment]
I'm merging this and will take a look at streaming

**Pull request merged**